item-impl/src/main/java/com/example/auction/item/impl/ItemRepository.java

```diff
@@ -34,7 +34,6 @@
     @Inject
     public ItemRepository(CassandraSession session, ReadSide readSide) {
         this.session = session;
-        session.underlying().thenAccept(s -> registerCodec(s, new EnumNameCodec<>(ItemStatus.class)));
```

Comment:
The codec is a client-side concern. It just handles mapping of objects into native Cassandra types. It does need to run on each node.